docs/kpod-tag.1.md

```diff
@@ -0,0 +1,33 @@
+% kpod(8) # kpod - Simple management tool for pods and images
+% Dan Walsh
```

Comment:
It looks like this and the other comment you made are only supposed to be in kpod.1.md, not the man pages for the individual subcommands, so I removed them.  Good catch!